tests/integration/ardrive.int.test.ts

```diff
@@ -70,6 +71,7 @@ describe('ArDrive class - integrated', () => {
 
 		// Declare common stubs
 		stub(walletDao, 'walletHasBalance').resolves(true);
+		stub(walletDao, 'getAddressWinstonBalance').resolves(stubbedEnoughBalance);
```

Comment:
We'll need to stub this only during sendAR tests since we'll put a boolean (with default false) to indicate whether or not to check wallet balance in the sendAR parameter list.